show/main.py

```diff
@@ -1557,6 +1558,7 @@ def interfaces():
                     neighbor_ip = bgp_peer[local_ip][1]
                 except Exception:
                     pass
+                neighbor_info.append([neighbor_name, neighbor_ip])
```

Comment:
Isn't it better to have the mapping based on the ipaddr (dictionary) instead of an implicit index? Just a thought!